Dockerfile

```diff
@@ -2,9 +2,26 @@ FROM golang:1.14.2-alpine3.11
 
 LABEL maintainer="The Mineiros.io Team <hello@mineiros.io>"
 
+# Terraform https://www.terraform.io/
 ENV TERRAFORM_VERSION=0.12.24
+ENV TERRAFORM_ARCHIVE=terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+ENV TERRAFORM_URL=https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/${TERRAFORM_ARCHIVE}
+ENV TERRAFORM_CHECKSUM=terraform_${TERRAFORM_VERSION}_SHA256SUMS
+ENV TERRAFORM_CHECKSUM_URL=https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/${TERRAFORM_CHECKSUM}
```

Comment:
do we need those as ENV or better ARGs? i guess this will be set in the actual environment of the running container this way?

Reply:
hmm .. this is only being used during build time so ARGS are the right way to go imho. We might switch to tfswitch, asdf, or so later in order to dynamically switch versions during runtime. wdyt @mariux ?

Reply:
changed in 3479ac